edx_solutions_api_integration/courses/serializers.py

```diff
@@ -46,7 +47,7 @@ def get_completion_percentage(self, obj):
         formats get completions as percentage
         """
         total_completions = self.context['total_completions'] or 0
-        completions = obj['completions'] or 0
+        completions = obj['earned'] or 0
         completion_percentage = 0
         if total_completions > 0:
             completion_percentage = min(int(round(100 * completions / float(total_completions))), 100)
```

Comment:
This value (given between 0 and 1, not 0 and 100) is also available as part of the api response (as `"percent"`, and does not need to be calculated here.